chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SearchViewController.java

```diff
@@ -497,4 +499,54 @@ public void download(@RequestParam(value="edition", required=false) String editi
 		return ddr;
 	}
 	
+	@ApiOperation(value="Get decertified certified products in the CHPL matching search criteria", 
+			notes="Returns up to all decertified certified products, their decertified statuses, and the total count of decertified certified products as the recordCount.")
+	@RequestMapping(value="/decertifications/certified_products", method=RequestMethod.GET,
+			produces="application/json; charset=utf-8")
+	public @ResponseBody SearchResponse getDecertifiedCertifiedProducts (
+			@RequestParam("searchTerm") String searchTerm, 
+			@RequestParam(value = "pageNumber", required = false) Integer pageNumber, 
+			@RequestParam(value = "pageSize", required = false) Integer pageSize,
+			@RequestParam(value = "orderBy", required = false) String orderBy,
+			@RequestParam(value = "sortDescending", required = false) Boolean sortDescending) throws EntityRetrievalException {
+		SearchResponse fullResp = new SearchResponse();
+		SearchResponse decertifiedCPs = new SearchResponse();
+		List<CertifiedProductSearchResult> cpResultList = new ArrayList<CertifiedProductSearchResult>();
+		
+		if (pageNumber == null){
+			pageNumber = 0;
+		}
+		
+		SearchRequest searchRequest = new SearchRequest();
+		

```

Comment:
Why don't we set the certificationstatuses filter of the searchRequest here? Then the DB could filter the results rather than having to put the logic in java below